CONTRIBUTING.md

```diff
@@ -1,53 +1,121 @@
-## How to Contribute
+## How to Contribute  
 
-1. Fork this repository to your github account.
+1. Fork this repository to your github account.  
 
 2. Clone the forked repository.  
-    ```
-    git clone https://github.com/<github-username>/STAC-IITMandi.github.io.git
-    ```
-3. Database Setup
+   `git clone https://github.com/<github-username>/STAC-IITMandi.github.io.git`  
 
-    1. You should have postgresql installed on your system. If not, run  
-        ```
-        sudo apt update
-        sudo apt install postgresql postgresql-contrib
-        ```
-    2. Create a local postgres database. Keep the credentials handy, you'll need them to link the database to django.
-    **Steps to create a postgresql db on your system**
-        1. `sudo su - postgres`
-        2. `psql`
-        3. `CREATE DATABASE <db-name>;`
-        4. `CREATE USER <db-username> WITH PASSWORD <db-password>;`
-        5. `ALTER ROLE <DB-USERNAME> SET client_encoding TO 'utf-8';`
-        6. `GRANT ALL PRIVILEGES ON DATABASE <db-name> TO <db-username>;`
-
-4. You can have the project running in a virtual environment or otherwise. Virtual environment is a preffered option.
-    **Steps to set up a virtual environment for this project**
-    1. `python3 -m venv myenv`
-
-    2. Activate the virtual environment `source myvenv/bin/activate`
-
-    3. Install required modules `pip install -r requirements.txt`
+3. Database Setup  
+   1. You should have postgresql installed on your system. If not, run    
+    
+      **On Linux** -    
+        `sudo apt update`  
+
+        `sudo apt install postgresql postgresql-contrib`  
+
+      **On Windows** -     
+            Install -     
+                  1. PostgreSQL - `https://www.postgresql.org/download/windows/`. 
+   
+                        **( Note :- Remeber Password entered here.)**.  
+                                 
+                  2. pgAdmin - `https://www.pgadmin.org/download/pgadmin-4-windows/`  
+                            
+
+
+   2. Create a local postgres database. Keep the credentials handy, you'll need them to link the database to django.  
+    **Steps to create a postgresql db on your system**  
+      **On linux** -  
+       1. `sudo su - postgres`  
+       2. `psql`  
+       3. CREATE DATABASE <db-name>;  
+       4. CREATE USER <db-username> WITH PASSWORD <db-password>;  
+       5. ALTER ROLE <DB-USERNAME> SET client_encoding TO 'utf-8';  
+       6. GRANT ALL PRIVILEGES ON DATABASE <db-name> TO <db-username>;    
```

Comment:
Use backticks for formatting.

Reply:
and enclose `<db-password>` in single quotes.

Reply:
done